.github/workflows/test.yml

```diff
@@ -85,7 +85,7 @@ jobs:
     name: Report coverage to Qlty
     runs-on: ubuntu-latest
     needs: test
-    if: success()
+    if: success() && (github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository)
```

Comment:
The secrets are not available for the qlty action for fork PRs, so would maybe just keep the current silent failure behavior